packages/documentation-site/patternfly-docs/pages/landing-pages/layouts/layouts-data.json

```
@@ -0,0 +1,23 @@
+{
+  "bullseye": {
+    "summary": "Bullseye layouts center content vertically and horizontally."
+  },
+  "flex": {
+    "summary": "Flex layouts create consistently spaced custom layouts that are responsive to other layouts and PatternFly components."
+  },
+  "gallery": {
+    "summary": "Gallery layouts arrange content in a responsive grid of uniform rows and columns."
+  },
+  "grid": {
+    "summary": "Grid layouts arrange content on a fixed-size 12 column grid."
+  },
+  "level": {
+    "summary": "Level layouts evenly distribute content horizontally."
+  },
+  "split": {
+    "summary": "Split layouts distribute multiple items horizontally, which adapt to viewport resizing."
+  },
+  "stack": {
+    "summary": "Stack layouts distribute multiple items vertically, which adapt to the available space of a container."
```

```suggestion
  "bullseye": {
    "summary": "Center content vertically and horizontally."
  },
  "flex": {
    "summary": "Create consistently spaced custom layouts that are responsive to other layouts and PatternFly components."
  },
  "gallery": {
    "summary": "Arrange content in a responsive grid of uniform rows and columns."
  },
  "grid": {
    "summary": "Arrange content on a fixed-size 12 column grid."
  },
  "level": {
    "summary": "Evenly distribute content horizontally."
  },
  "split": {
    "summary": "Distribute multiple items horizontally, which adapt to viewport resizing."
  },
  "stack": {
    "summary": "Distribute multiple items vertically, which adapt to the available space of a container."
```

What about removing the name of each layout to avoid redundancy.